### PR TITLE
[TensorIf] add a NULL checker

### PR DESCRIPTION
**Pull request by helloahn**
To improve code quality, I want to suggest a null checker.

Signed-off-by: Hyoung Joo Ahn <hello.ahn@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

**Review by dongju-chae — APPROVED** [review]
LGTM

File: gst/nnstreamer/tensor_if/gsttensorif.c [hunk 1]
```diff
@@ -370,6 +370,12 @@ gst_tensor_if_set_property_supplied_value (const GValue * value,
 {
   gint i;
   gboolean is_float = FALSE;
+
+  if (!value) {
+    ml_loge ("Invalid supplied value. The value is NULL");
+    return;
+  }
```
Comment (wooksong) [inline gst/nnstreamer/tensor_if/gsttensorif.c:377]: As far as I know, gnu89 allows mixed declarations and code. However, in my environment, this breaks the build. Please check it out.
```
$ ninja -C build
ninja: Entering directory `build'
[41/233] Compiling C object 'gst/nnstreamer/73d9e45@@nnstreamer@sha/gsttensorif.c.o'.
FAILED: gst/nnstreamer/73d9e45@@nnstreamer@sha/gsttensorif.c.o 
cc -Igst/nnstreamer/73d9e45@@nnstreamer@sha -Igst/nnstreamer -I../gst/nnstreamer -Igst/nnstreamer/./include/ -I../gst/nnstreamer/./include/ -Igst/nnstreamer/tensor_transform -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/gstreamer-1.0 -I/usr/include/orc-0.4 -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=gnu89 -g '-DVERSION="1.7.1"' '-DVERSION_MAJOR="1"' '-DVERSION_MINOR="7"' '-DVERSION_MICRO="1"' -Wwrite-strings -Wformat -Wformat-nonliteral -Wformat-security -Winit-self -Waddress -Wno-multichar -Wvla -Wpointer-arith -Wmissing-declarations -Wmissing-include-dirs -Wmissing-prototypes -Wnested-externs -Waggregate-return -Wold-style-definition -Wdeclaration-after-statement '-DNNSTREAMER_CONF_FILE="etc/nnstreamer.ini"' -DENABLE_TENSORFLOW=1 -DENABLE_TENSORFLOW_LITE=1 -DHAVE_ORC=1 -DENABLE_FLATBUF=1 -Wredundant-decls -fPIC -pthread -MD -MQ 'gst/nnstreamer/73d9e45@@nnstreamer@sha/gsttensorif.c.o' -MF 'gst/nnstreamer/73d9e45@@nnstreamer@sha/gsttensorif.c.o.d' -o 'gst/nnstreamer/73d9e45@@nnstreamer@sha/gsttensorif.c.o' -c /home/wook/Work/NNS/tmp/nnstreamer/gst/nnstreamer/tensor_if/gsttensorif.c
/home/wook/Work/NNS/tmp/nnstreamer/gst/nnstreamer/tensor_if/gsttensorif.c: In function ‘gst_tensor_if_set_property_supplied_value’:
/home/wook/Work/NNS/tmp/nnstreamer/gst/nnstreamer/tensor_if/gsttensorif.c:379:3: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
   const gchar *param = g_value_get_string (value);
   ^~~~~
cc1: all warnings being treated as errors
[50/233] Compiling C object 'gst/nnstreamer/73d9e45@@nnstreamer@sta/tensor_converter.c.o'.
ninja: build stopped: subcommand failed.
```

File: gst/nnstreamer/tensor_if/gsttensorif.c [hunk 2]
```diff
@@ -370,6 +370,12 @@ gst_tensor_if_set_property_supplied_value (const GValue * value,
 {
   gint i;
   gboolean is_float = FALSE;
+
+  if (!value) {
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_if/gsttensorif.c:374]: Whether value is null or not seems to be checked by g_value_get_string internal.
Do we need this?

Comment (helloahn) [reply gst/nnstreamer/tensor_if/gsttensorif.c:374]: If we `use g_value_get_string()` like below, it generates error.
```c
  const gchar *param = g_value_get_string (NULL);
```
```bash
(process:23753): GLib-GObject-CRITICAL **: 13:39:03.002: g_value_get_string: assertion 'G_VALUE_HOLDS_STRING (value)' failed
```

File: gst/nnstreamer/tensor_if/gsttensorif.c (hunk 1 above)
Comment (helloahn) [reply gst/nnstreamer/tensor_if/gsttensorif.c:380]: it's fixed :)

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by again4you — APPROVED** [review]
LGTM!

**Review by zhoonit — APPROVED** [review]
LGTM :)

**Review by wooksong — APPROVED** [review]

**Pull request merged**